components/select/src/MultipleSelect.svelte

```diff
@@ -118,10 +126,6 @@
       window.removeEventListener("click", onHide);
     };
   });
-
-  // ensure that active option is not out of bounds
-  $: if (activeOptionIdx < 0 || activeOptionIdx > filteredOptions.length - 1)
-    activeOptionIdx = 0;
```

Comment:
this is needed, because after filtering the active option index might be out of bounds, so need to reset to 0

Reply:
yes this is handled in the switch cases if you notice

Reply:
cause reactive statement will run everytime when we reference the variable. so its better if we can reduce this by handling it on demand on the switch case

Reply:
if so, should set `activeOptionIdx = 0` before return